src/main/groovy/net/wasdev/wlp/gradle/plugins/LibertySingleServerTasks.groovy

```diff
@@ -47,140 +28,65 @@ class LibertySingleServerTasks extends LibertyTasks {
     }
 
     void applyTasks() {
-        overwriteTask('compileJSP', CompileJSPTask, {
-            description 'Compile the JSP files in the src/main/webapp directory. '
-            logging.level = LogLevel.INFO
+        project.compileJSP {
             dependsOn 'installLiberty', 'compileJava'
-            group 'Liberty'
-        })
-
-        overwriteTask('installLiberty', InstallLibertyTask, {
```

Comment:
Why did alot of these tasks get removed? installLiberty, libertyStop, libertyDump etc.

Reply:
They didn't have additional configuration that needed to be added later in the build.

Reply:
I moved the some properties that were getting set in that file to the individual task files. For some of the tasks those are the only things we were setting so we don't need to configure any additional properties at that point.